src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -282,16 +341,43 @@ public static boolean isItemSimilar(@Nullable ItemStack item, @Nullable ItemStac
         } else if (checkAmount && item.getAmount() < sfitem.getAmount()) {
             return false;
         } else if (sfitem instanceof SlimefunItemStack && item instanceof SlimefunItemStack) {
-            return ((SlimefunItemStack) item).getItemId().equals(((SlimefunItemStack) sfitem).getItemId());
+            SlimefunItemStack stackOne = (SlimefunItemStack) sfitem;
+            SlimefunItemStack stackTwo = (SlimefunItemStack) item;
```

Comment:
The variables aren't needed since we can just use the instanceof and add the variable there.

Reply:
Ha yeah this PR was from before we added patterns into SF lol - showing how old it is. Ill push something later today